OpenRA.Mods.Common/Lint/CheckTranslationReference.cs

```diff
@@ -27,19 +28,61 @@ class CheckTranslationReference : ILintPass
 		readonly Dictionary<string, string[]> referencedVariablesPerKey = new Dictionary<string, string[]>();
 		readonly List<string> variableReferences = new List<string>();
 
-		public void Run(Action<string> emitError, Action<string> emitWarning, ModData modData)
+		void ILintPass.Run(Action<string> emitError, Action<string> emitWarning, ModData modData)
 		{
-			var language = "en";
+			var language = "en"; // TODO
```

Comment:
Is this a stray or forgotten TODO? Please add a description to it if it's a valid TODO.

Reply:
the TODO here is as in this isn't supposed to be hardcoded

Reply:
I could just remove the TODO and merge this. It's fairly obvious that language shouldn't be hardcoded

Reply:
I don't think it's obvious at when reading just this part of the code. Perhaps it's OK to have it hard-coded at "en" because that's all we need? Or is it? Better make it clear if there's anything that needs further attention in this part of the code and was deliberately not implemented.

Reply:
```suggestion
			// TODO: Check all available translation languages
			var language = "en";
```
?